silverpeas/content.md

```diff
@@ -0,0 +1,179 @@
+# What is Silverpeas
+
+[Silverpeas](https://www.silverpeas.org) is a Collaborative and Social-Networking Portal built to facilitate and to leverage the collaboration, the knowledge-sharing and the feedback of persons, teams and organizations.
+
+Accessible from a simple web browser or from a smartphone, Silverpeas is used every days by ourselves. With about 30 ready-to-use applications and with its conversional and relational functions, it makes possible for users to work together, share their knowledge and good practices, and in general to improve their reciprocal empathy, therefore their willingness to collaborate.
+
+Silverpeas is usually used as an intranet and extranet platform dedicated to collaboration and information sharing.
+
+%%LOGO%%
+
+# How to use this image
+
+Docker images of Silverpeas require one of the following database system in order to be used:
+
+-	the open-source, powerful and recommended PostgreSQL database system,
+-	the Microsoft SQLServer database system,
+-	the Oracle database system.
+
+The Silverpeas images support actually only the two first database systems; because of the non-free licensing issues with the Oracle JDBC drivers, Silverpeas cannot include these drivers by default and consequently it cannot use transparently an Oracle database system as a persistence backend.
+
+For the same reasons, the Docker images of Silverpeas aren't shipped with the Oracle JVM but with the default OpenJDK. Silverpeas uses the Wildfly application server as runtime.
+
+The Silverpeas images use the following environment variables to set the database access parameters:
+
+-	`DB_SERVERTYPE` to specify the database system to use with Silverpeas: POSTGRESQL for PostgreSQL, MSSQL for Microsoft SQLServer, ORACLE for Oracle. By default, it is set to POSTGRESQL.
+-	`DB_SERVER` to specify the IP address or the name of the host on which the database system is running. By default, it is set to `localhost`.
```

Comment:
It's probably worth considering changing this default from `localhost` to `db` or `database` or something so that either Docker networks or container linking (`--link some-postgres:database`) can work out-of-the-box (especially given that looking for `localhost` in a container doesn't make much sense).